pkg/internal/mock/vdri/mock_registry.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package vdri
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+	vdriapi "github.com/hyperledger/aries-framework-go/pkg/framework/aries/api/vdri"
+)
+
+// MockVDRIRegistry mock implementation of vdri
+// to be used only for unit tests
+type MockVDRIRegistry struct {
```

Comment:
Same as above, we should expose mocks from aries.

Reply:
done